_i18n/nb-no/resources/user-guides/multisig-messaging-system.md

```diff
@@ -0,0 +1,864 @@
+
+## Innholdsfortegnelse
+- [Innholdsfortegnelse](#table-of-contents)
+- [Introduksjon](#introduction)
+- [Monero Multisig i et nøtteskall](#monero-multisig-in-a-nutshell)
+- [Arkitekturen til MMS](#the-architecture-of-the-mms)
+- [Brukeropplevelsen til MMS](#the-mms-user-experience)
+  - [Et meldingssystem](#a-messaging-system)
+  - [Signerere og meldinger](#signers-and-messages)
+- [Å få tak i MMS](#getting-the-mms)
+- [Installasjon og konfigurering av PyBitmessage](#installing-and-configuring-pybitmessage)
+- [Ytterligere PyBitmessage-justeringer](#further-pybitmessage-tweaks)
+- [MMS-kommandooversikt](#mms-command-overview)
+- [Konfigurering av lommebok for bruk med MMS](#configuring-a-wallet-for-use-with-the-mms)
+  - [Adresser og merkelapper](#addresses-and-labels)
+  - [Å kjøre CLI-lommeboken](#running-cli-wallet)
+  - [Initialisering av MMS](#initializing-the-mms)
+  - [Konfigurering av signerere](#configuring-signers)
+  - [Manuell konfigurering av signerere](#manually-configuring-signers)
+  - [Auto-Config](#auto-config)
+  - [Sending av signererkonfigurasjon](#sending-signer-configuration)
+- [Etablering av Multisig-adresse](#establishing-the-multisig-address)
+- [Finansiering av Multisig-lommeboken](#funding-the-multisig-wallet)
+- [Synkronisering av lommebøker](#syncing-wallets)
+- [Å foreta Multisig-transaksjoner](#making-multisig-transactions)
+- [Kommandoene i detalj](#the-commands-in-detail)
+  - [mms init](#mms-init)
+  - [mms info](#mms-info)
+  - [mms signer](#mms-signer)
+  - [mms list](#mms-list)
+  - [mms next](#mms-next)
+  - [mms sync](#mms-sync)
+  - [mms transfer](#mms-transfer)
+  - [mms delete](#mms-delete)
+  - [mms send](#mms-send)
+  - [mms receive](#mms-receive)
+  - [mms note](#mms-note)
+  - [mms show](#mms-show)
+  - [mms export](#mms-export)
+  - [mms set](#mms-set)
+  - [mms start\_auto\_config](#mms-start_auto_config)
+  - [mms auto\_config](#mms-auto_config)
+  - [mms stop\_auto\_config](#mms-stop_auto_config)
+  - [mms send\_signer\_config](#mms-send_signer_config)
+- [Sikkerhet](#security)
+  - [Bruk av kryptering og signaturer](#use-of-encryption-and-signatures)
+  - [CKommunikasjon mellom MMS og PyBitmessage](#communication-mms-to-pybitmessage)
+  - [Etterligning](#impersonation)
+  - [Angrepskontrollerte data](#attacker-controlled-data)
+- [Feilsøking](#troubleshooting)
+  - [Å løse synkroniseringsproblemer](#solving-syncing-troubles)
+  - [Omdirigering av en transaksjon til en annen signerer](#redirecting-a-transaction-to-another-signer)
+  - [Ignorering av usamarbeidsvillige signerere under synkronisering](#ignoring-uncooperative-signers-when-syncing)
+  - [Gjenoppretting av tapte eller dupliserte meldinger](#recovering-from-lost-or-duplicate-messages)
+  - [Korrigering/oppdatering av signererinformasjon](#correcting--updating-signer-information)
+  - [Å starte fra bunn](#starting-from-scratch)
+  - [MMS-/PyBitmessage-interaksjoner](#mms--pybitmessage-interactions)

```

Comment:
The links in the TOC should be adapted to the localized version.

For example `[Introduksjon](#introduction)` should be `[Introduksjon](#introduksjon)`

Reply:
Fixed. I have assumed the three Norwegian letters `æ`, `ø`, `å` are allowed to use in these tags. If not, please let me know, and I'll replace them.